runtime/onert/core/include/ir/operation/BCQFullyConnected.h

```diff
@@ -37,7 +37,8 @@ class BCQFullyConnected : public Operation
     INPUT = 0,
     WEIGHTS_SCALES,
     WEIGHTS_BINARY,
-    BIAS
+    BIAS,
+    WEIGHTS_CLUSTER,
```

Comment:
Actually, I am receiving as **CLUSTERS**, not `CLUSTER` :(

Reply:
I agree. It's fixed.